### PR TITLE
[CU-3kj0mhy] Fix get report

### PR DESCRIPTION
**Pull request by RonnyChiang**
* fix: getReportPhotos controller & service & model
         增加 filter params. 僅能填入 statusMap裡有的key (no_issue, pending, deleted)
* feat: getReportsPhotos test

### CODE REVIEW

**Comment by WendellLiu** [comment]
Task linked: [CU-3kj0mhy fix get report be filter by status](https://app.clickup.com/t/3kj0mhy)

File: src/services/report-service/index.js [hunk 1]
```diff
@@ -77,6 +77,11 @@ class ReportService extends BaseService {
     }
   }
   async getReportPhotos(paginationOption) {
+    // check filter word
+    if (
+      !Object.prototype.hasOwnProperty.call(statusMap, paginationOption.filter) // https://eslint.org/docs/latest/rules/no-prototype-builtins
```
Comment (WendellLiu) [inline src/services/report-service/index.js:82]: 為什麼要這麼複雜的方式來判斷有沒有在map 中？

File: src/services/report-service/index.js [hunk 2]
```diff
@@ -77,6 +77,11 @@ class ReportService extends BaseService {
...
+    )
+      throw new GeneralError(reportErrorMap['wrongReportStatusType']);
```
Comment (WendellLiu) [inline src/services/report-service/index.js:84]: 我覺得就性質而言，錯誤不是一個好主意

**Review by WendellLiu — CHANGES_REQUESTED** [review]

File: src/services/report-service/index.js (hunk 1 above)
Comment (RonnyChiang) [reply src/services/report-service/index.js:82]: !(paginationOption.filter in statusMap) ?

**Review by WendellLiu — APPROVED** [review]

**Pull request merged**